### ISSUE TITLE
Question: What is the difference between this and checkiz/elixir-mongo?

### ISSUE THREAD

**Issue by MarkNijhof** [elixir-mongo/mongodb#49]
I see this one is started about a year ago while checkiz/elixir-mongo is a lot older. What should one chose when starting fresh and for what reason? Thanks for your feedback :-)

**Comment by ericmj** [comment]
First off, It's best if you make a comparison yourself and make a decision from that because I am obviously objective but I can explain the reasons why I created this library instead of using elixir-mongo. Also note that the reasons I mention may very well be fixed in elixir-mongo today.

At first i was looking at using elixir-mongo but I needed to add features and fix some bugs, unfortunately I felt that the quality of the code was lacking so I was not comfortable working on it. There were so many things for me to fix I didn't have time to go through the process of PRs. So instead of working on improving elixir-mongo I thought it was better to create a new library from scratch.

I also thought documentation was lacking, it didn't follow the client library specification that 10gen has created, it had bugs which could cause duplicate ObjectIds to be generated among other things. I also believe that nowadays this library has more features.
